documentation/performance/attribute-memory-usage.html

```diff
@@ -267,6 +259,16 @@ <h3 id="variants">Variants</h3>
       <td>Document vector, Multi-value mapping, Enum store, Posting list</td>
       <td>D * IW * ROF + D * V * EW * ROF + U * (FW+VW+4) * ROF + D * V * PW * ROF</td>
     </tr>
+    <tr>
+      <td>Boolean single-value</td>
+      <td>Document vector</td>
+      <td>D * FW * ROF</td>
+    </tr>
+    <tr>
+      <td>Boolean multi-value</td>
```

Comment:
I think this still should be "Numeric multi-value value plain"?

Reply:
not sure what you mean? I added bool single / multi value - and removed a duplicate entry for _Numeric multi-value value plain_ at (see https://docs.vespa.ai/documentation/performance/attribute-memory-usage.html)